engine/src/main/java/org/terasology/rendering/nui/layers/mainMenu/videoSettings/DisplayModeSetting.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2016 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.rendering.nui.layers.mainMenu.videoSettings;
+
+import static java.lang.System.in;
+
+/**
+ */
+public enum DisplayModeSetting {
```

Comment:
This could use some javadoc imho. You should shortly mention what the display modes are (take it from the discussions about borderless/fullscreen/...).